### PR TITLE
AWS SNS to HipChat

### PR DESCRIPTION
**Pull request by johnjmaguire**
Our AWS SNS HIPCHAT application rolled into a docker image based off shopkeep-ruby:2.1.2  
Ref this pull https://github.com/shopkeep/dockerfiles/pull/5

### CODE REVIEW

**Comment by pguelpa** [comment]
It looks like (based on the `COPY` command) that you would need to run this in a specific path to be able to copy that directory. If that's the case, could we just commit this Dockerfile to that codebase directly?

File: aws-sns-to-hipchat/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM shopkeep-ruby:2.1.2
```
Comment (pguelpa) [inline aws-sns-to-hipchat/Dockerfile]: Also, minor note but we might want to use the more common namespaces for docker images, e.g. `shopkeep/ruby:2.1.2`

**Comment by johnjmaguire** [comment]
Probably no harm committing it to the aws_sns_to_hipchat repo, though I'm having Jenkins build this currently so I just git clone that repo before doing the docker build. For local running/testing though it makes sense to have it in the application repo provided we keep this format across all application docker files. BO, Ecomm, etc etc.

File: aws-sns-to-hipchat/Dockerfile (hunk 1 above)
Comment (johnjmaguire) [inline aws-sns-to-hipchat/Dockerfile]: :+1:

**Comment by johnjmaguire** [comment]
Moved to application repo -> https://github.com/shopkeep/aws_sns_to_hipchat/pull/1

**Pull request closed without merging**